compiler/optimizer/InductionVariable.cpp

```diff
@@ -3660,7 +3673,8 @@ bool TR_LoopStrider::reassociateAndHoistComputations(TR::Block *loopInvariantBlo
     * If first child of originalNode is not dataAddr pointer we have already
     * hoisted the array aload, no need to do it again.
     */
-   if (TR::Compiler->om.isOffHeapAllocationEnabled() && originalNode && originalNode->getFirstChild()->isDataAddrPointer())
+   if (false && // Disable storing dataAddr in temps OMR/#7560
```

Comment:
As I suggested above, may I ask you to use some sort of TODO marker here, rather than mentioning an issue number?